clientSide/src/components/DropDown/DropDown.css

```diff
@@ -0,0 +1,18 @@
+.drop {
+  border: hidden;
+  box-shadow: 3px 4px 7px var(--global-black);
```

Comment:
Nice one, I think I will make this box-shadow global to use it somewhere else as well, since we also have it as a variable on figma <3